docs/golden-retriever.mdx

```diff
@@ -32,6 +32,21 @@ If restore is unsuccessful for certain files, they will be marked as “ghosts
 the Dashboard UI, and a message + button offering to re-select those files will
 be displayed.
 
+<details>
+  <summary>How much data can Golden Retriever store?</summary>
+
+The amount of storage is dependent upon how much available disk space the device
+has. The general range is between 50MB and 20GB.
+
+| Disk Space | Storage Limit                     |
+| ---------- | --------------------------------- |
+| < 8GB      | 50MB                              |
+| > 8-32GB   | 500MB                             |
+| > 32-128GB | 4% of volume size                 |
+| > 128GB    | 4% or 20GB (whichever is smaller) |
```

Comment:
Should we link to MDN or some other source, in case this changes for instance?

Reply:
There is no single source of truth link, the amount available to service worker caching isn't defined in the specification. The amount varies by browser depending on device and storage conditions. This is a generalised version of the differences and it unlikely to change.

Reply:
Found this, wouldn't hurt? https://developer.mozilla.org/en-US/docs/Web/API/Storage_API/Storage_quotas_and_eviction_criteria

Reply:
Wow I missed that one, that's actually quite good